tests-cuda/test_1300_same_for_numba_cuda.py

```diff
@@ -0,0 +1,1382 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+import numpy as np
+import pytest
+
+import awkward as ak
+
+try:
+    import numba as nb
+    import numba.cuda as nb_cuda
+    from numba import config
+
+    ak.numba.register_and_check()
+
+except ImportError:
+    nb = nb_cuda = None
+
+numbatest = pytest.mark.skipif(
+    nb is None or nb_cuda is None, reason="requires the numba and numba.cuda packages"
+)
```

Comment:
This can also be done with

```suggestion
nb = pytest.importorskip("numba")
nb_cuda = pytest.importorskip("numba.cuda")
pytest.importorskip("numba.config")
config = nb.config

try:
    ak.numba.register_and_check()
except ImportError:
    pytest.skip(
        reason="too old Numba version", allow_module_level=True
    )
```

Also, pytest skips can happen at module-level, not just function-level.

Reply:
Note: you can batch all of these suggested edits into a single commit on the "Files changed" tab.